docs/codeql/codeql-language-guides/customizing-library-models-for-go.rst

```diff
@@ -107,15 +107,15 @@ The remaining values are used to define the ``access path``, the ``kind``, and t
 - The eighth value ``sql-injection`` is the kind of the sink. The sink kind is used to define the queries where the sink is in scope. In this case - the SQL injection queries.
 - The ninth value ``manual`` is the provenance of the sink, which is used to identify the origin of the sink.
 
-Example: Taint source from the ``net`` package
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-This example shows how the Go query pack models the return value from the ``Listen`` method as a ``remote`` source.
-This is the ``Listen`` function which is located in the ``net`` package.
+Example: Taint source from the ``net/http`` package
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+This example shows how the Go query pack models the return value from the ``FormValue`` method as a ``remote`` source.
+This is the ``FormValue`` method of the ``Request`` struct which is located in the ``net/http`` package.
```

Comment:
```suggestion
This is the ``FormValue`` method of the ``Request`` type which is located in the ``net/http`` package.
```
I can't quite put my finger on why, but it sounds odd to talk about it as a struct. I guess the official documentation doesn't do that, and I've got used to that. (Any named/defined type can have methods defined on it.)